foundationdb/src/subspace.rs

```diff
@@ -103,21 +106,21 @@ mod tests {
 
     #[test]
     fn sub() {
-        let ss0 = Subspace::new(&(1,));
-        let ss1 = ss0.subspace(&(2,));
+        let ss0: Subspace = (1,).into();
```

Comment:
It's not a scope of this PR, but is it okay to convert non-tuple element to single-element tuple implicitly? For example now following example results on same subspace
```
let ss1: Subspace = (1,).into();
let ss2: Subspace = 1.into();
assert_eq!(ss1.bytes(), ss1.bytes());
```

If it makes sense, we can make this line shorter with `1.into()`

Reply:
That's already possible. As long as the element implements `Encode` it can converted. Let me clean up the tests...